bigspicy.py

```diff
@@ -447,8 +53,11 @@ def DefineOptions(optparser):
   optparser.add_option('--verilog_include', dest='verilog_includes', default=[], action='append', help='verilog include paths')
   optparser.add_option('--verilog_defines', dest='verilog_defines', default=[], action='append', help='verilog macro definitions')
   optparser.add_option('--spef', dest='spef_files', default=[], action='append', help='spef files')
+  
+  # FIXME: what is the difference between these two "spice" options? 
```

Comment:
--spice files are read for their contents (i.e. a circuit.Module is loaded from their netlists)
--spice_headers are read for their port orders and kept as ExternalModules